### ISSUE TITLE
Enhancement: Remove Expressions Tool also to remove corresponding effect?

### ISSUE THREAD

**Issue by reneandritsch** [Rainbox-dev/DuAEF_Duik#372]
### Tool & Version
- Duik 16.1.0-Beta

### After Effects
- CC2018 (15.1.2)
- English

### OS
- Mac OS 10.13

# What's the issue?
When using the “Remove Expressions” Tool, effect stays on the layer and has to be removed manually. It would speed up the workflow if the corresponding effect would be removed with the expression if that is possible.

**Comment by Nico-Duduf** [comment]
Yes, I've been wanting a remove button for the Duik tools for a long time and never had time to implement it.
I did not thought this remove expression too could do it easily!
